lib/util.js

```diff
@@ -25,6 +25,33 @@ export const async = {
             fns[i](cb);
         }
     },
+    /**
+     * Attempts to run all funcs and callbacks positively on the first success.
+     * @param  {Function[]} fns
+     * @param  {Function} callback
+     */
+    some(fns, callback) {
+        let todo = fns.length;
+        let firstFailed;
+        const cb = (err, res) => {
+            if (err) {
+                if (!firstFailed) {
+                    firstFailed = err;
+                }
+                todo--;
+                if (todo === 0) {
+                    return callback(firstFailed);
+                }
+                return null;
+            }
+            todo = 0;
+            return callback(null, res);
+        };
+
+        for (let i = 0; i < fns.length; i++) {
```

Comment:
This could be optimized by having `i < fns.length && todo > 0`, which will halt subsequent runs after we get any function that works. Since most optimizations are synchronous, this could lend some speed to .alternatives